cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/service/GcpResourceNameService.java

```diff
@@ -21,13 +23,22 @@ public class GcpResourceNameService extends CloudbreakResourceNameService {
 
     private static final int ATTACHED_DISKS_PART_COUNT = 4;
 
+    private static final int INSTANCE_GROUP_PART_COUNT = 3;
+
     private static final int INSTANCE_NAME_PART_COUNT = 3;
 
     private static final int LOAD_BALANCER_WITH_PORT_NAME_PART_COUNT = 3;
 
     @Value("${cb.max.gcp.resource.name.length:}")
     private int maxResourceNameLength;
 
+    public GcpResourceNameService(int maxResourceNameLength) {
+        this.maxResourceNameLength = maxResourceNameLength;
+    }
+
+    public GcpResourceNameService() {
+    }
+
```

Comment:
so this field is only available via reflection therefore is impossible to test with unit test without spring context. I removed this but developers should be aware - this makes coverage worse since we cannot check if decode works with encode - exactly the case why the bug occurred.

Reply:
![image](https://user-images.githubusercontent.com/1827293/211507125-c92feebd-7a6d-46ba-af0e-a9d3637885c1.png)

Reply:
You can find examples of using ReflectionUtils without Spring context to test this kind of class where parameter values are injected.

Reply:
I choose to rely on other verification instead